docs/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -0,0 +1,118 @@
+[role="xpack"]
+[[monitoring-metricbeat]]
+=== Monitoring {kib} with {metricbeat}
+
+beta[] In 6.4 and later, you can use {metricbeat} to collect and ship data about 
+{kib}, rather than routing it through {es} as described in <<monitoring-kibana>>. 
+
+. Enable the collection of monitoring data in the {stack}. Set 
+`xpack.monitoring.collection.enabled` to `true` on the production cluster. +
++ 
+--
+For example, you can use the following APIs to review and change this setting:
+
+[source,js]
+----------------------------------
+GET _cluster/settings
+
+PUT _cluster/settings
+{
+  "persistent": {
+    "xpack.monitoring.collection.enabled": true
+  }
+}
+----------------------------------
+
+For more information, see {ref}/monitoring-settings.html[Monitoring settings in {es}] 
+and {ref}/cluster-update-settings.html[Cluster update settings].
+--
+
+. Disable the default collection of {kib} monitoring metrics. +
++
+--
+Add the following setting in the `kibana.yml` file on your production cluster: 
+
+[source,yaml]
+----------------------------------
+xpack.monitoring.kibana.collection.enabled: false
+----------------------------------
+
+Leave the `xpack.monitoring.enabled` set to its default value (`true`). 
+
+For more information, see 
+<<monitoring-settings-kb,Monitoring settings in {kib}>>.
+--
+
+. {metricbeat-ref}/metricbeat-installation.html[Install {metricbeat}] on the 
+same node as {kib}.
+
+. Enable the {kib} module in {metricbeat}. +
++
+--
+For example, to enable the default configuration in the `modules.d` directory, 
+run the following command:
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+metricbeat modules enable kibana
+----------------------------------------------------------------------
+
+Alternatively, you can enable module configs in the `metricbeat.yml` file. For 
+more information, see 
+{metricbeat-ref}/configuration-metricbeat.html[Specify which modules to run] and 
+{metricbeat-ref}/metricbeat-module-kibana.html[{kib} module]. 
+--
+
+. Configure the {kib} module in {metricbeat}. +
++
+--
+For example, you can change the default module configuration by modifying the 
```

Comment:
I think the wording ought to be a bit more prescriptive here. In order for things to work correctly with metricbeat monitoring kibana, users need to have the exact configuration in `modules.d/kibana.yml` as shown in the code snippet below.